brain_friendly.py

```diff
@@ -85,6 +86,17 @@ def get_brace_matches(program):
     else:
         return brace_pairs
 
+
+def eval_file(filename, index, memory, input=None, output=None):
+    f = open(filename, 'r')
+    program = f.read()
+    f.close()
```

Comment:
Change this to
```python
with open(filename, 'r') as f:
    program = f.read()
```
since it automatically handles closing the file, and is prettier.